packages/cli/README.md

```diff
@@ -0,0 +1,65 @@
+# B7 CLI tool
+
+We have created a simple CLI tool for everyday tasks on Bichard. We have used a library called [Commander.js](https://github.com/tj/commander.js)
+
+To install the CLI tool please run the command `npm run install:global` this will install the CLI tool globally.
+
+If you want to do some development work on this tool, you can run the watch command `npm run install:watch`
+
+To invoke the tool you the command is `b7`
+
+```
+CLI tool for Bichard 7
+
+Options:
+  -V, --version    	output the version number
+  --e2e            	Use the e2e environment
+  --uat            	Use the uat environment
+  --preprod        	Use the preprod environment
+  --prod           	Use the production environment
+  --shared         	Use the shared environment
+  -h, --help       	display help for command
+
+Commands:
+  status [options] 	Get healthcheck endpoint output from production
+  dev-sgs          	Apply dev security groups to out environments
+  fetch-image <image>  Fetch images from AWS Elastic Container Repository
+  message-processing   EventBridge rule that forwards incoming messages through to Conductor
+  conductor        	Bichard workflow engine
+  cloudwatch       	A method for querying cloudwatch for user events
+  help [command]   	display help for command)
+```
+
+Each subcommand has its own help docs you can provide the `-h or --help` flag.
+e.g `b7 dev-sgs --help`
+
+```
+b7 dev-sgs --help
+Usage: b7 dev-sgs [options]
+
+Apply dev security groups to out environments
+
+Options:
+  -h, --help  display help for command
+
+```
+
+## Querying cloudwatch
+
+A user query that comes up time and time again is that a user can log into Bichard. We would previously have to log into AWS and check container logs for user activity. On most occasions the user would of provided the incorrect email that we have stored in Postgres.
+Now we can easily query our User service and postfix logs with a users email.
```

Comment:
```suggestion
A common user query is that a user cannot log into Bichard. This is typically because the user has used an email address that doesn't match what we have stored in the database.

Previously, we had to log into AWS and check container logs for user activity but now we can easily query our User service and postfix logs with a users email.
```